app/models/combination.rb

```diff
@@ -57,8 +57,8 @@
 #
 class Combination < TaxonName
 
-  # The ranks that can be used to build combinations. ! TODO:  family group names  ?
-  APPLICABLE_RANKS = %w{family subfamily tribe subtribe genus subgenus section subsection
```

Comment:
Verified in production, looks safe to remove:
```ruby
2.7.2 :001 > ap TaxonNameRelationship.where(TaxonNameRelationship.arel_table[:type].matches('TaxonNameRelationship::Combination::%')).select(:type).distinct(:type)
[
    [0] #<TaxonNameRelationship::Combination::Genus:0x000056394f003910> {
          "id" => nil,
        "type" => "TaxonNameRelationship::Combination::Genus"
    },
    [1] #<TaxonNameRelationship::Combination::Subgenus:0x000056394ef590f0> {
          "id" => nil,
        "type" => "TaxonNameRelationship::Combination::Subgenus"
    },
    [2] #<TaxonNameRelationship::Combination::Species:0x000056394ef00108> {
          "id" => nil,
        "type" => "TaxonNameRelationship::Combination::Species"
    },
    [3] #<TaxonNameRelationship::Combination::Subspecies:0x000056394eea70d0> {
          "id" => nil,
        "type" => "TaxonNameRelationship::Combination::Subspecies"
    },
    [4] #<TaxonNameRelationship::Combination::Variety:0x000056394ee4e0e8> {
          "id" => nil,
        "type" => "TaxonNameRelationship::Combination::Variety"
    }
]
 => nil 
```

Reply:
Thanks for the check.